input/pagecontent/authentication.md

```diff
@@ -0,0 +1,174 @@
+
+Verifiable credentials have a long lifetime, up to several years. To be able to revoke a verifiable credential, a revocation mechanism is needed. The chosen revocation mechanism is [StatusList 2021](https://w3c-ccg.github.io/vc-status-list/), which defines a standard for revoking verifiable credentials using a bitstring. The verifier periodically retrieves (and caches) the status list and verifies the existence of the credential in the status list.
+
+#### Summary technologies, and standards per transaction
+
+| Transaction   | Technology / Standard                                                                                                                           | Description                                                                                                                                                                                     |
+| ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| GFI-001       | [DID (Decentralized Identifier) 1.0](https://www.w3.org/TR/did-1.0/)                                                                            | DID method did:web, a domain name based identifier that hosts the DID document containing the public key                                                                                        |
+| GFI-002       | [OpenID Connect for Verifiable Credential Issuance 1.0 (OpenID4VCI)](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html) | Protocol to request and issue verifiable credentials between digital agents and authoritative registries                                                                                        |
+| GFI-00\[2,4\] | [Verifiable Credentials (VC) 1.1](https://www.w3.org/TR/2022/REC-vc-data-model-20220303/)                                                       | Standard for expressing identity claims in a cryptographically verifiable way                                                                                                                   |
+| GFI-00\[2,4\] | [Verifiable Presentations (VP) 1.1](https://www.w3.org/TR/2022/REC-vc-data-model-20220303/#presentations-0)                                     | Standard for presenting a set of verifiable credentials in a cryptographically verifiable way                                                                                                   |
```

Comment:
Now I think of it, shouldn't we aim for VC data model v2? Or have a growth path?

Reply:
Can we support 2.0 during the POCs phase? I don't thing that is possible now. What we can do is create profiles of compatible specs which we can use in a growth path. Made an issue: https://github.com/nuts-foundation/nl-generic-functions-ig/issues/61